nodebuilder/p2p/module.go

```diff
@@ -58,7 +58,7 @@ func ConstructModule(tp node.Type, cfg *Config) fx.Option {
 			"p2p",
 			baseComponents,
 			fx.Provide(blockstoreFromDatastore),
-			fx.Provide(func(ctx context.Context, bootstrappers Bootstrappers) (network.ResourceManager, error) {
+			fx.Provide(func(ctx context.Context, bootstrappers Bootstrappers) ([]ma.Multiaddr, error) {
```

Comment:
We usually define type with a name that would say what's this about instead of just returning Multiaddr, as the could mean anything